### ISSUE TITLE
RSS Feed/API access of incoming requests 

### ISSUE THREAD

**Issue by and** [IEEEKeralaSection/rescuekerala#627]
### RSS Feed/API access of requests
Can we release incoming requests data as an RSS feed or as an api? 
This can be used to:
1. Move incoming requests to another tracker or to a service like ifttt.com to trigger some event. 
2. For others to build on top of this data like: #617

**Comment by and** [comment]
Just saw a thread about ticketing system - #20

**Comment by and** [comment]
For future reference. The data endpoint:
https://keralarescue.in/data
(down as of now)

**Comment by lunaroyster** [comment]
Yeah the /data endpoint wasn't going to scale. It was serving an excessively large file.

I hope the replacement is built soon, since I was making use of the data at https://kerala-crowdsource.itsarnavb.me. I was being super careful (caching the data, and updating only once an hour) but alas, /data is dead

**Comment by and** [comment]
How does the pagination work?
https://keralarescue.in/data/?page=1
https://keralarescue.in/data/?page=2
etc.. gives the same result.
